app/class_room.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :lable
+
+  @students = []
```

Comment:
You have done great initializing the has-many relationship between classroom and student. Kindly define this within the constructor in order to be initialized for every object of the class and also for there not to be errors.